server/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "server_name" {
+  description = "The name of the Transfer server"
+  type        = string
+  default     = "sftp-server"
+}
+
+variable "vpc_id" {
+  description = "The id of the vpc for the endpoint for the Transfer server"
+  type        = string
+}
+
+variable "aws_region" {
+  description = "The region for the Transfer server"
+  type        = string
+  default     = "us-east-1"
+}
```

Comment:
This should be derived from the calling module. You can get this through a data call like this:

```
data "aws_region" "current" {}
```